src/main/java/com/dev/cinema/Main.java

```diff
@@ -76,7 +78,7 @@ public static void main(String[] args) throws AuthenticationException {
 
         OrderService orderService = (OrderService) injector.getInstance(OrderService.class);
         orderService.completeOrder(shoppingCartService.getByUser(iryna).getTickets(), iryna);
-        System.out.println(shoppingCartService.getByUser(iryna));
-        System.out.println(orderService.getOrderHistory(iryna));
+        logger.info("Get order by user id: " + shoppingCartService.getByUser(iryna));
+        logger.info("Get order by user id: " + orderService.getOrderHistory(iryna));
```

Comment:
let's wrap call to `login()` method in try-catch and add log.warn with the informative message in `catch` block